.circleci/config.yml

```diff
@@ -95,16 +95,17 @@ commands:
           name: Building docker image for << parameters.target >>
           command: |
             build_tag="${CIRCLE_SHA1}"
-            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}"
-            docker build -t ${DOCKHUB_ORGANISATION}/binary-static-bot:${build_tag} -t ${DOCKHUB_ORGANISATION}/binary-static-bot:latest .
+            build_latest="latest"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}" && build_latest="beta-latest"
+            docker build -t ${DOCKHUB_ORGANISATION}/binary-static-bot:${build_tag} -t ${DOCKHUB_ORGANISATION}/binary-static-bot:build_latest .
       - run:
           name: Pushing Image to docker hub
           command: |
             build_tag="${CIRCLE_SHA1}"
-            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}" && build_latest="beta-latest"
             echo $DOCKERHUB_PASSWORD | docker login -u $DOCKERHUB_USERNAME --password-stdin
             docker push ${DOCKHUB_ORGANISATION}/binary-static-bot:${build_tag}
-            docker push ${DOCKHUB_ORGANISATION}/binary-static-bot:latest
+            docker push ${DOCKHUB_ORGANISATION}/binary-static-bot:build_latest
```

Comment:
It's better to remove `tag` in `docker push` command or change `build_latest` as a variable
```
docker push ${DOCKHUB_ORGANISATION}/binary-static-bot
```
or 
```
docker push ${DOCKHUB_ORGANISATION}/binary-static-bot:${build_latest}
```